04_HowTos/DataReduction/DRAGONS_reduction_examples/NIRI_Imaging_Supernova/NIRI_Imaging_Supernova.ipynb

```diff
@@ -453,7 +457,9 @@
     "\n",
     "The master dark and flat will be retrieved automatically from the local calibration database. Again, the user BPM needs to be specified on the command line.\n",
     "\n",
-    "The output stack units are in electrons (header keyword BUNIT=electrons). The output stack is stored in a multi-extension FITS (MEF) file. The science signal is in the \"SCI\" extension, the variance is in the \"VAR\" extension, and the data quality plane (mask) is in the \"DQ\" extension."
+    "The output stack units are in electrons (header keyword BUNIT=electrons). The output stack is stored in a multi-extension FITS (MEF) file. The science signal is in the \"SCI\" extension, the variance is in the \"VAR\" extension, and the data quality plane (mask) is in the \"DQ\" extension.\n",
+    "\n",
+    "The final reduced image will be saved as N20160102S0296_image.fits."
```

Comment:
Change
_"The final reduced image will be saved as N20160102S0296\_image.fits."_
with
_"The final reduced image will be saved with the suffix \_image.fits."_